pkg/trtl/trtl.go

```diff
@@ -131,13 +169,36 @@ func (h *TrtlService) Put(ctx context.Context, in *pb.PutRequest) (out *pb.PutRe
 		out.Meta = returnMeta(object)
 	}
 
+	// Compute Put latency in milliseconds
+	latency := float64(time.Since(start)/1000) / 1000.0
+	pmLatency.WithLabelValues("Put").Observe(latency)
+
+	// Increment prometheus Put counter
+	pmPuts.WithLabelValues(ns).Inc()
+
+	// TODO: prometheus
+	// If in.Options.ReturnMeta is true, we will get metadata from honu
+	// Unfortunately, we currently we don't get tombstone information, but if we did,
+	// we could use it to decrement our `pmTombstone` counter here
+
```

Comment:
Put in this TODO but wanted to call it out here -- can you think of any way we can get the tombstones without fiddling with `honu`?

Reply:
I see what you mean, unfortunately, the only thing I can think of is to do an `h.db.Object` before the `Put` to see if the current version is a Tombstone - however, because we don't have transactions this introduces the possibility of an inconsistency if two concurrent puts see a tombstone and decrement the tombstone counter twice instead of once.

The right way would be to change Honu to have it indicate if the parent version was a Tombstone or not; but that does require a change to Honu. I think this `TODO` is the best we can do for now.